pkgs/tools/security/onioncircuits/default.nix

```
@@ -10,8 +10,8 @@ python3.pkgs.buildPythonApplication rec {
     sha256 = "13mqif9b9iajpkrl9ijspdnvy82kxhprxd5mw3njk68rcn4z2pcm";
   };
 
-  nativeBuildInputs = [ intltool ];
-  buildInputs = [ intltool gtk3 gobject-introspection ];
+  nativeBuildInputs = [ pkg-config intltool ];
```

```suggestion
  nativeBuildInputs = [ intltool ];
```

Pasted the wrong thing